### ISSUE TITLE
Refactor previous code for testing orientation changes with the custom function written for it [Blocked on Landscape mode]

### ISSUE THREAD

**Issue by PrarabdhGarg** [oppia/oppia-android#682]
There are a lot of test files that use the espresso methods directly to change orientation while testing the app, even though a custom method exists for the same. 
Basically there are different types of orientation change related test cases, which have bee implemented in different ways. But we need all of them to be consistent and all of them should use only `orientationLandscape()` function to do orientation change.

The file that needs refactoring are:

- [x] `TopicPracticeFragmentTest.kt`
- [ ] `InputInteractionViewTestActivity.kt`
- [x] `StateFragmentTest.kt`
- [x] `AudioFragmentTest.kt`
- [x] `ConceptCardFragmentTest.kt`
- [x] `TopicLessonsFragmentTest.kt`
- [x] `TopicReviewFragmentTest.kt`
- [x] `TopicInfoFragmentTest.kt`

**Comment by PrarabdhGarg** [comment]
@rt4914 can I work on this after my current issue is solved?

**Comment by rt4914** [comment]
@PrarabdhGarg Nice pick. You can definitely work on this. But I have an alternate proposal too. Because you know this solution very well, can we assign this task to someone new and you can guide them to finish this, similar to what you and me are doing right now. How does this sound?

**Comment by PrarabdhGarg** [comment]
Perfect. I would love to do that

**Comment by rt4914** [comment]
> Perfect. I would love to do that

Great. Then we will assign it to someone new.

**Comment by rt4914** [comment]
> @PrarabdhGarg Nice pick. You can definitely work on this. But I have an alternate proposal too. Because you know this solution very well, can we assign this task to someone new and you can guide them to finish this, similar to what you and me are doing right now. How does this sound?

cc @seanlip

**Comment by chrk2205** [comment]
@PrarabdhGarg @rt4914 what exactly should we do in this??
i would like to work on this

**Comment by rt4914** [comment]
> @PrarabdhGarg @rt4914 what exactly should we do in this??
> i would like to work on this

Currently we cannot work on this as the title suggests that it is blocked on landscape mode.

**Comment by rt4914** [comment]
> @PrarabdhGarg @rt4914 what exactly should we do in this??
> i would like to work on this

Also, checkout our [wiki](https://github.com/oppia/oppia-android/wiki) first. This will help you in setting up the project and once you sign the CLA and fill the survey form we can assign you issues to work on.

**Comment by chrk2205** [comment]
@rt4914  
I already signed the CLA and set my project up.

**Comment by rt4914** [comment]
> @rt4914
> I already signed the CLA and set my project up.

You haven't filled the survey form. Please fill that too.

**Comment by chrk2205** [comment]
Sorry,
Done. @rt4914

**Comment by rt4914** [comment]
Assigned to @chrk2205

**Comment by PrarabdhGarg** [comment]
@chrk2205 has your doubt been resolved, or do you still have some doubt in this issue?

**Comment by chrk2205** [comment]
@PrarabdhGarg 
All I have to do is override Oncofiguration method and call orientation landscape method . Is it? 
Did I miss something ??
Thanks

**Comment by PrarabdhGarg** [comment]
@chrk2205 you don't have to override anything.
You have to replace 
`activityScenario.onActivity { activity ->
      activity.requestedOrientation = Configuration.ORIENTATION_LANDSCAPE
    }`
code with the custom defined `orientationLandscape()` function, as used in `ConceptCardFragmentTest.kt`

**Comment by rt4914** [comment]
@chrk2205 I think this PR will create issues for you based on the other PRs which are in progress right now. I will be assigning you one new issue to get started.

**Comment by chrk2205** [comment]
@rt491 okay.

**Comment by misterjedu** [comment]
@rt4914  my first PR just got accepted and merged. 

Can I work on this as my second?

**Comment by rt4914** [comment]
> @rt4914 my first PR just got accepted and merged.
> 
> Can I work on this as my second?

@misterjedu Assigned.

**Comment by misterjedu** [comment]
@rt4914  and @BenHenning , I just checked all the files listed above and it seems they've already been fixed a long time ago.  The orientationLandscape() function is already being used to change orientation while testing the app.

**Comment by rt4914** [comment]
> @rt4914 and @BenHenning , I just checked all the files listed above and it seems they've already been fixed a long time ago. The orientationLandscape() function is already being used to change orientation while testing the app.

@misterjedu Thanks for checking. I think only `InputInteractionViewTestActivity` file needs changes for this issue to get completed. But all those test cases are ignored as of now. 

Also, it has been mentioned in the title that it is blocked on landscape mode. I will deassign this issue as of now. Thanks.

**Comment by adhiamboperes** [comment]
@rt4914, what does it mean for this issue to be blocked on landscape mode? It has been a while; has the status changed, and can we now assign someone to work on this?

**Comment by BenHenning** [comment]
@adhiamboperes it's been a while since I've thought about this issue, but I suspect it's at least blocked by #2572 (and in fact it appears that PR was planned to address this issue). It's a bit tough to wrap up that work which is why I haven't been able to get back to it.

**Comment by adhiamboperes** [comment]
Thanks @BenHenning. We'll leave this in the backlog for now.
